ocaml/libs/tracing/tracing.ml

```diff
@@ -620,6 +631,15 @@ module Export = struct
           span_list
       with e -> debug "Tracing: ERROR %s" (Printexc.to_string e)
 
+    let flush_spans () =
+      let span_list = Spans.since () in
+      get_tracer_providers ()
+      |> List.filter (fun x -> x.TracerProvider.enabled)
+      |> List.iter (fun x ->
```

Comment:
To avoid the nested iter, concat_map can be used:
```ocaml
get_tracer_providers ()
|> List.filter (fun x -> x.TracerProvider.enabled)
|> List.concat_map (fun x -> Traceprovider.endpooints.of x)
|> List.iter (export_to_endpoint span_list)
```